client/post-editor/editor-revisions-list/style.scss

```diff
@@ -0,0 +1,59 @@
+.editor-revisions-list__date {
+	font-weight: bold;
+}
+
+.editor-revisions-list__changes {
+	font-size: 11px;
+}
+
+.editor-revisions-list__additions {
+	color: $blue-medium;
+}
+
+.editor-revisions-list__deletions {
+	color: $alert-red;
+}
+
+.editor-revisions-list__list,
+.editor-revisions-list__header {
+	background: $white;
+	border-top: 1px solid darken( $sidebar-bg-color, 5% );
+	border-bottom: 1px solid darken( $sidebar-bg-color, 5% );
+	font-size: 13px;
+}
+
+.editor-revisions-list__revision {
+	border-top: 1px solid darken( $sidebar-bg-color, 5% );
+
+	&.selected {
+		background: $blue-medium;
+
+		&, *, .button {
```

Comment:
To "force" a white color with a quite heavy specificity, there are a couple of ways out of "*", none are worth it imho:
- `&, span, .button` -> `span` is (almost) as generic and carries a toned down message than: "let's white all the thingz!" which is really what we want here :) 
- `&, .editor-revisions-list__deletions, .editor-revisions-list__additions, .button, ... ` (with `...` containing a lot of classes) this will constantly break each time the markup is updated.
- `&, .button` + a few blocks like this:

```
.editor-revision-list__deletions {
    .is-selected & {
        color: $white
    }
}
```
This introduces considerably more code and makes it harder to change the `is-selected` style for everything (though we can minimize that if we anticipate to only change the color with: `$revision-selected-color: $white`)